bicycleparameters/main.py

```diff
@@ -896,7 +896,7 @@ def eig(self, speeds):
 
     def plot_eigenvalues_vs_speed(self, speeds, fig=None, generic=False,
                                   color='black', show=False, largest=False,
-                                  linestyle='-', grid=False):
+                                  linestyle='-', grid=False, showLegend=True):
```

Comment:
Let's make this `show_legend` to follow PEP8. I wrote much of this code before I knew about PEP8 and new things should adopted PEP8. I'll eventually update all the code to follow.

Reply:
Be sure to add this in the docstring below too.

Reply:
Okay, by docstring did you mean the if statement? I think I have everything corrected now.

Reply:
Yeah that's what it is, I went ahead and added my own description there, go ahead and change it if you would like.

Reply:
> The comment area below the function explaining the parameters?

Yes.